ui/e2e/src/tests/variables.spec.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { test, expect } from '../fixtures/dashboardTest';
+
+test.use({
+  dashboardName: 'Variables',
+  modifiesDashboard: true,
+});
+
+test.describe('Dashboard: Variables', () => {
+  test('can add simple text variable', async ({ dashboardPage }) => {
+    await dashboardPage.startEditing();
+    await dashboardPage.startEditingVariables();
+
+    const variableEditor = dashboardPage.getVariableEditor();
+    await variableEditor.addVariable();
```

Comment:
reading this now, `addVariable` makes me think an entire variable just got added

Could do `startAddingVariable` to follow the pattern you have with other functions?

Reply:
Good call! Fixing it now.